server/ai_http.go

```diff
@@ -801,3 +804,29 @@ func parseMultiPartResult(body io.Reader, boundary string, pipeline string) core
 
 	return wkrResult
 }
+
+// handleStreamStatus returns the latest available status of a live-video-to-video stream
+func (h *lphttp) handleStreamStatus() http.HandlerFunc {
+	return func(w http.ResponseWriter, r *http.Request) {
+		if r.Method != http.MethodGet {
+			http.Error(w, "Method not allowed", http.StatusMethodNotAllowed)
+			return
+		}
+
+		streamID := r.URL.Query().Get("stream")
+		if streamID == "" {
+			respondWithError(w, "stream parameter is required", http.StatusBadRequest)
+			return
+		}
+
+		// Get status for specific stream
+		status, exists := monitor.GetStreamStatus(streamID)
+		if !exists {
+			respondWithError(w, "Stream status not found", http.StatusNotFound)
+			return
+		}
+
+		jsonData, _ := json.Marshal(status)
```

Comment:
I know it's very rare, but preferrably do not ignore this error, return a 500 if you get sth